crime-application-tracking/build.gradle

```diff
@@ -34,18 +35,20 @@ dependencies {
 
 	implementation "io.sentry:sentry-spring-boot-starter"
 	implementation "io.sentry:sentry-logback"
-
 	implementation 'org.springframework.boot:spring-boot-starter-actuator'
-	//implementation 'org.springframework.boot:spring-boot-starter-security'
 	implementation 'org.springframework.boot:spring-boot-starter-validation'
 	implementation 'org.springframework.boot:spring-boot-starter-web'
 	implementation 'org.springframework.boot:spring-boot-starter-webflux'
+	implementation "org.springframework.boot:spring-boot-starter-oauth2-resource-server"
+
+	implementation "uk.gov.justice.service.laa-crime:crime-commons-spring-boot-starter-rest-client:$versions.crimeCommonsVersion"
```

Comment:
I think we can remove this dependency as we are not using , please verify.

Reply:
It's used in the CrimeApplicationTrackingTestConfiguration class.